src/components/common/transactions-list/transactions-list.component.coffee

```diff
@@ -206,7 +206,14 @@ module.component('transactionsList', {
 
     # Remove all children and update parent
     deleteTransactionsGroup = (trx) ->
-      trxGroupId = if trx.recurring then trx.id else trx.recurring_parent
+      if trx.recurring
+        trxGroupId = trx.id
+        trx.recurring = false
+      else
+        return unless trx.recurring_parent
+        trxGroupId = trx.recurring_parent
```

Comment:
Shouldn't be `trx.recurring_parent.id` here?

Reply:
Actually no because `trx.recurring_parent` is already id... the name is not good... should be `trx.recurring_parent_id`. I agree.. but the resource in Bolt is like it... :/